### ISSUE TITLE
Merit mode does not disable when switching to a job >75

### ISSUE THREAD

**Issue by GanimanSwift** [project-topaz/topaz#1367]
**_I have:_**

- [x] searched existing issues (http://project-topaz.com/issues/) to see if the issue has already been opened
- [x] checked the commit log to see if the issue has been resolved since my server was last updated

**_Additional Information_** (Steps to reproduce/Expected behavior) **:** 

On a level 75 capped server, switch to a level 75 job. Go to the Status menu and enable merit mode. Switch to a job under level 75. Check status. Job is "locked" and colored blue. XP is greyed out.

I have not tested if you actually earn merit points with a job under level 75, but you cannot switch modes back to exp without first switching back to a level 75 job.

Edit: This effect persists after zoning.

**Comment by TeoTwawki** [comment]
does it persist even after your newly set job changes zones? I ask because my hunch is we fail to send an update packet or something silly like that

**Comment by GanimanSwift** [comment]
> does it persist even after your newly set job changes zones? I ask because my hunch is we fail to send an update packet or something silly like that

Yes, it persists after zoning. Forgot to mention, had confirmed before submitting.
